docs/user-guide/cli-using-global-profile-configuration.md

```diff
@@ -166,7 +166,13 @@ You can continue to add more LPARs and more services within each LPAR. After you
 
 When you first run the `zowe config init --global-config` command, the `profiles.base.properties.user` and `profiles.base.properties.password` fields are defined to the "secure" array in your configuration file, which help to ensure that the username and password are stored securely on your computer.
 
-To store or update values for the secure fields (for example, when you want to change your username and password), issue the `zowe config secure` command.
+To store or update values for the secure fields (for example, when you want to change your username and password), issue the `zowe config secure` command. If, for example, you want to update several property values in a long list of properties, press `Enter` to skip a field.
+
+**Example:**
+
+```
+Enter profiles.base.properties.user - blank to skip:
+```
```

Comment:
Not sure whether the example is necessary. Perhaps we could just say something like:
> If you want to update only a few property values in a long list of properties, press `Enter` to skip entering values for properties you don't want to update.

@MikeBauerCA Thoughts?

Reply:
@JamesBauman I agree with Timothy that I don't think an example is needed here. Otherwise, looks good to me.